source/administration/changelog.md

```diff
@@ -9,9 +9,7 @@ Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 Mattermost v5.17.0 contains medium to high level security fixes. [Upgrading](http://docs.mattermost.com/administration/upgrade.html) is recommended. Details will be posted on our [security updates page](https://about.mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://www.mattermost.org/responsible-disclosure-policy/).
 
 - **v5.17.1, release day TBD**
-  - Fixing an issue where deactivating a user should increment the daily and monthly active users counts down (not up) in the System Console. [MM-19957](https://mattermost.atlassian.net/browse/MM-19957)
-  - Fixing an issue where guest users are not removed from a team if removed from all channels. [MM-20153](https://mattermost.atlassian.net/browse/MM-20153)
-  - Fixing an issue where leaving a channel does not work in some cases if the channel is open in another webapp client. [MM-20206](https://mattermost.atlassian.net/browse/MM-20206)
+  - Fixing an issue where leaving a channel does not work in some cases if the channel is open in another webapp or desktop client. [MM-20206](https://mattermost.atlassian.net/browse/MM-20206)
```

Comment:
@amyblais I think there was a CSM asking when 5.17.1 is shipping -- while we don't know the date, you can suggest to refer the customer to the changelog (or share the open issue) to determine if it's something they want to wait for

Reply:
I know. That is why I'm updating the changelog so that it has up-to date information.